mango_base/mango/dashboards/time_series_utils/constants.py

```diff
@@ -0,0 +1,40 @@
+# Constants for time aggregation
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez